<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GearBoxScenePlayer</class>
 <widget class="QWidget" name="GearBoxScenePlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GearBoxScenePlayer</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="playButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../GearBox.qrc">
         <normaloff>:/GearBox/icons/iconfinder_Music_media_player_ui_button_play_6613197.png</normaloff>:/GearBox/icons/iconfinder_Music_media_player_ui_button_play_6613197.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../GearBox.qrc">
         <normaloff>:/GearBox/icons/iconfinder_Music_media_player_ui_button_stop_6613187.png</normaloff>:/GearBox/icons/iconfinder_Music_media_player_ui_button_stop_6613187.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reloadScriptButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>76</width>
         <height>72</height>
        </size>
       </property>
       <property name="text">
        <string>Reload Scripts</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="renderingAPILabel">
       <property name="text">
        <string>Rendering API: Unknown</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="gpuDeviceLabel">
       <property name="text">
        <string>GPU Device: Unknown</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../GearBox.qrc"/>
 </resources>
 <connections/>
</ui>
